ui/src/components/testing/TestingPage.js

```diff
@@ -0,0 +1,51 @@
+import {
+    Box,
+    Container,
+    VStack,
+    Text,
+    Button,
+    HStack,
+    Image,
+    Link,
+  } from '@chakra-ui/react';
+  
+  const TestingPage = () => {
+  
+    return (
+      <Box bgColor="#F6FAFF">
+        <Container
+          maxW="container.xl"
+          paddingTop={5}
+          paddingBottom={{ base: '4rem', lg: '5rem' }}
+          fontFamily="DMSerifDisplay_Italic"
+        >
+          <Box
+            max-width="100%"
+            display="flex"
+            flexDir={'column'}
+            alignItems={'center'}
+            gap={{ base: '1.75rem', md: '2.5rem', lg: '4rem' }}
+          >
+            <VStack
+              maxWidth={{ base: '95%', lg: '100%' }}
+              spacing={{ base: '1rem', lg: '1.5rem' }}
+            >
+              <Box mb="5" alignSelf="flex-start">
+                <Text textStyle={'heading.1'}
+                >
+                  Discover HMCC of Hong Kong
+                </Text>
+                <Text textStyle={'heading.4.bold'}
+                >
```

Comment:
[Good to have]: know that this is the suggested official documentation for custom themes. But wondering if there are ways to make chakra show all the possible custom theme, like `heading.1` `heading.2` etc, similar to MUI `useTheme()`. Mostly to make our life much easier lol